### DISCUSSION TITLE
guestbook/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#8] [Announcements]
# guestbook/

互联网的广大朋友们，欢迎光临我的小博客！欢迎留言！

https://www.haoyep.com/guestbook/

**Comment by congcongi123** [comment]
你的内容真的很不错呢～

**Comment by shawnfinelee** [comment]
老板，你这个用的什么博客系统搭建的这么清爽

**Reply by leegical** [reply]
- 静态博客编译工具hugo
- 主题[Fixit](https://fixit.lruihao.cn/zh-cn/documentation/getting-started/quick-start/)
- 本博客的大致搭建过程可以参考[这篇文章](https://www.haoyep.com/posts/windows-hugo-blog-github/)

**Comment by huqianshan** [comment]
zotero的教程很完善！ 但是我添加了自动跳转的宏脚本，还是无法变成超链接。

**Comment by zhang17173** [comment]
博客做的很棒！！！

**Comment by yushou81** [comment]
我是一个学了快两年计算机还是一头雾水的小白（比较菜），但是大大的文章居然让我也看懂了。太感谢了！！！终于配置好zsh了

**Comment by KuthorX** [comment]


**Reply by leegical** [reply]
你的博客貌似没有友链页面呢

**Comment by BenChuat** [comment]
想问问这个简体-繁体-英文连文章内容都变了是怎么实现的？

**Comment by manchun20** [comment]
感谢大大的教程，发现了宝藏，就是不知为什么网页图片加载不出来

**Comment by dayoo2346** [comment]
感谢！教程对小白很友好！

**Reply by manchun20** [reply]
刚才收到一封邮件，顺着收件信息一路找到这里，才知道原来所有的评论都会自动抄送给参与讨论的人。很高兴能通过这种方式遇到志趣相投的人。
